packages/common/selectors/entities/team.js

```diff
@@ -30,7 +30,10 @@ export const teamSortedListSelector = createSelector(
   teams => {
     const defaultTeam = teams.find(team => team.type === TEAM_TYPE.DEFAULT);
     const otherTeams = teams
-      .filter(team => team.type !== TEAM_TYPE.DEFAULT)
+      .filter(
+        team =>
+          team.type !== TEAM_TYPE.DEFAULT && team.type !== TEAM_TYPE.PERSONAL,
```

Comment:
Also you can use `include` for ability of code extending :smile: 
```
[TEAM_TYPE.DEFAULT, TEAM_TYPE.PERSONAL].includes(team.type)
```

Reply:
✔️